go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/drone-plugins/drone-plugin-lib
 go 1.19
 
 require (
+	github.com/pkg/errors v0.9.1
```

Comment:
Please don’t add this package back. Use normal go error wrapping.

Reply:
This is already merged but what I was suggesting was to use the error wrapping that was added in Go 1.13 which made the errors package obsolete. See https://go.dev/blog/go1.13-errors